example/example/tests/test_grapple.py

```diff
@@ -825,7 +825,10 @@ def test_query_url_field_with_direct_document_serve_method(self):
         settings.WAGTAILDOCS_SERVE_METHOD = serve_method_at_test_start
 
     def tearDown(self):
-        self.example_document.file.delete()
+        file = self.example_document.file
+        self.example_document.delete()
+        file.close()
+        file.delete(save=False)
```

Comment:
could this be reduced to:

```
file = self.example_document.file
file.close()
file.delete()
```
I don't use Windows, so cannot test.

Reply:
I tried and that didn't work. But while verifying I was getting in consistent results and looked more closely at the test. I found the file was being accessed without a context manager, so I wrapped the file access in a context manager instead.